inc/cleantalk-public-integrations.php

```diff
@@ -2108,15 +2108,50 @@ function apbct_form__ninjaForms__testSpam()
 
     $checkjs = apbct_js_test(Sanitize::cleanTextField(Cookie::get('ct_checkjs')), true);
 
-    /**
-     * Filter for POST
-     */
-    $input_array = apply_filters('apbct__filter_post', $_POST);
+    $form_data = json_decode(TT::toString(Post::get('formData')), true);
+    if ( ! $form_data ) {
+        $form_data = json_decode(stripslashes(TT::toString(Post::get('formData'))), true);
+    }
+    if ( function_exists('Ninja_Forms') && isset($form_data['fields']) ) {
+        /** @psalm-suppress UndefinedFunction */
+        $nf_form_fields_info = Ninja_Forms()->form()->get_fields();
```

Comment:
What if Ninja Forms update changed the get_fields output? Or the methods will be renamed?

Probably, we do need try-catch there.

Reply:
Done https://github.com/CleanTalk/wordpress-antispam/pull/462/commits/674ed801322a9399cd5840b644fa6662ddf2c87c